templates/README.md

```diff
@@ -0,0 +1,68 @@
+Topic name
+==========
+
+*Short introduction about the topic, why it is important to learn it (the questions behind this topic), what are the objectives, ...*
+
```

Comment:
I added time estimation in the tutorial template and update the image in `CONTRIBUTING`